spec/services/platform/submitter_payload_spec.rb

```diff
@@ -0,0 +1,183 @@
+    described_class.new(service: service, user_data: user_data)
+  end
+
+  let(:user_data) do
+    {
+      'name_text_1' => 'Legolas',
+      'email-address_email_1' => 'legolas@middle.earth.com',
+      'parent-name_text_1' => 'Thranduil',
+      'your-age_number_1' => '2931',
+      'family-hobbies_text_1' => 'Archery',
+      'do-you-like-star-wars_radios_1' => 'Only on weekends',
+      'holiday_date_1(3i)' => '30',
+      'holiday_date_1(2i)' => '12',
+      'holiday_date_1(1i)' => '2020',
+      'burgers_checkboxes_1' => ['Beef, cheese, tomato', 'Chicken, cheese, tomato']
+    }
+  end
+  let(:pdf_heading) do
+    "Middle Earth characters"
+  end
+  let(:pdf_subheading) do
+    "Gossip magazine about middle earth characters"
+  end
+  let(:email_to) do
+    "middle.earth.entertainment@magazine.co.uk"
+  end
+  let(:email_from) do
+    "MoJ forms <moj-online@digital.justice.gov.uk>"
+  end
+  let(:email_subject) do
+    'All info about middle earth characters'
+  end
+  let(:email_body) do
+    'Please find attached Elfs info!'
+  end
+
+  let(:pages_payload) do
+    [
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "name_text_1",
+            field_name: "Full name",
+            answer: "Legolas"
+          }
+        ]
+      },
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "email-address_email_1",
+            field_name: "Your email address",
+            answer: "legolas@middle.earth.com"
+          }
+        ]
+      },
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "parent-name_text_1",
+            field_name: "Parent name",
+            answer: "Thranduil"
+          }
+        ]
+      },
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "your-age_number_1",
+            field_name: "Your age",
+            answer: "2931"
+          }
+        ]
+      },
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "family-hobbies_text_1",
+            field_name: "Your family hobbies",
+            answer: "Archery"
+          }
+        ]
+      },
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "do-you-like-star-wars_radios_1",
+            field_name: "Do you like Star Wars?",
+            answer: "Only on weekends"
+          }
+        ]
+      },
+      {
+        heading: "",
+        answers: [
+          {
+            field_id: "holiday_date_1",
+            field_name: "What is the day that you like to take holidays?",
+            answer: "30 December 2020"
+          }
+        ]
+      },
+      {
+        heading: '',
+        answers: [
+          {
+            field_id: "burgers_checkboxes_1",
+            field_name: "What would you like on your burger?",
+            answer: ['Beef, cheese, tomato', 'Chicken, cheese, tomato']
+          }
+        ]
+      }
+    ]
```

Comment:
No tests for multiple questions yet. See comment above

Reply:
also, we can wait for multiple questions to be supported